code/style.css

```diff
@@ -1,7 +1,240 @@
+}
+.form,
+.accordion,
+.greeting {
+	justify-content: flex-start;
+	padding: 10px 20px 20px 20px;
+	width: min(80%, 320px);
+	border: 1px solid #ffd25d;
+	border-radius: 5px;
+	background-color: #ffd25d;
+	box-shadow: 0px 5px 50px 5px slategray;
+}
+
+/* form styling */
+.form {
+	align-items: center;
+}
+
+.text-field-container {
+	display: flex;
+	justify-content: space-between;
+	flex-wrap: wrap;
+	width: 100%;
+}
+.text-field {
+	width: 100%;
+	background: transparent;
+	height: 1.8rem;
+	padding: 1px 5px 1px 30px;
+	font-size: 0.9rem;
+	border-radius: 4px 4px 4px 4px;
+	border: 1px solid #f9b44c;
+}
+
+input[type='text'] {
+	background: url('./images/user.png') no-repeat 8px 50% #e8f0fd;
+	background-size: 15px;
+}
+input[type='email'] {
+	background: url('./images/email.png') no-repeat 8px 50% #e8f0fd;
+	background-size: 15px;
+}
+input[type='password'] {
+	background: url('./images/padlock.png') no-repeat 8px 50% #e8f0fd;
+	background-size: 15px;
+}
+.validation-text {
+	display: none;
+	padding-right: 20px;
+	font-size: 0.8rem;
+	color: red;
+}
+input:invalid ~ label {
+	display: inline;
+}
+input:invalid {
+	border: 1px solid red;
+}
+input:valid {
+	border: 1px solid limegreen;
+}
+.radio-container {
+	flex-wrap: wrap;
+	flex-direction: column;
+}
+.checkbox-container {
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-start;
+}
+input[type='radio'] {
+	width: 10%;
+}
+.radio {
+	width: 50vw;
+}
+button {
+	box-sizing: border-box;
+	width: 100%;
+	border-radius: 4px 4px 4px 4px;
+	border: 1px solid #f9b44c;
+	background-color: #e8f0fd;
+	height: 2rem;
+	padding: 1px 30px;
+	font-size: 0.9rem;
+}
+form:invalid > button {
+	color: grey;
+}
 
+/* styling of accordion */
+.accordion h1 {
+	text-align: center;
+}
+.question {
+	margin-top: 3px;
+	padding: 5px 5px 5px 20px;
+	border-radius: 4px;
+	background-color: #e8f0fd;
+	border: 1px solid #f9b44c;
+}
+.answer {
+	display: none;
+}
+.icon-arrow-right {
+	position: absolute;
+	margin: 3px 0px 0px -17px;
+	height: 0.9rem;
+}
+.active > .icon-arrow-right {
+	transform: rotate(90deg);
+	margin: 5px 0px 0px -16px;
+}
+.answer p {
+	margin: 0 0 2px 0;
+	padding: 10px 10px;
+	border-radius: 0px 0px 4px 4px;
+	background-color: #ffffff;
+	font-size: 0.9rem;
+	box-shadow: inset 0 0 4px #5c5c5c;
+	border-bottom: 1px solid #f9b44c;
+	border-right: 1px solid #f9b44c;
+	border-left: 1px solid #f9b44c;
+}
 .active + .answer {
-  /* some styling here */
+	display: inherit;
+	animation: growDown 1s ease-in-out forwards;
+	transform-origin: top center;
+}
+.question.active {
+	border-radius: 4px 4px 0px 0px;
+	border-top: 1px solid #f9b44c;
+	border-right: 1px solid #f9b44c;
+	border-left: 1px solid #f9b44c;
+	border-bottom: 0px;
+}
+.question:hover,
+.radio:hover,
+input[type='radio']:hover,
+.checkbox:hover,
+input[type='checkbox']:hover {
+	cursor: pointer;
+}
+.greeting-container {
+	display: none;
+	text-align: center;
+}
+
+footer {
+	height: 10px;
+}
+
+@keyframes growDown {
```

Comment:
Maybe this could be explained a little bit further? I haven't seen the `@keyframes growDown` before.